trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid/module_persistence_levelpool_hybrid.F

```diff
@@ -0,0 +1,371 @@
+! This module defines and instantiates objects
+! for a hybrid persistence levelpool type
+! reservoir. The hybrid reservoir type
+! inherits input and output stucts from the
+! reservoir base module and calls instantiation
+! of these into sub-objects. The hybrid
+! reservoir type also points to types for
+! hybrid parameters and state and calls
+! instantiation of these into sub-objects.
+! A pointer to a levelpool reservoir object
+! is also held in state, and this module
+! instantiates that levelpool object. There
+! is also a subroutine to run hybrid reservoir
+! that is derived from the reservoir base
+! type interface to run reservoir.
+
+module module_persistence_levelpool_hybrid
+
+    use module_persistence_levelpool_hybrid_parameters, only: hybrid_parameters_interface
+    use module_persistence_levelpool_hybrid_state, only: hybrid_state_interface
+    use module_levelpool, only: levelpool_type
+    use module_reservoir_utilities, only: modify_for_projected_storage, warn_negative_inflow, &
+                                          create_hybrid_diagnostic_log_file, &
+                                          log_hybrid_diagnostic_data
+    use module_reservoir_base
```

Comment:
can we use `only` with this (module_reservoir_base) and in module_levelpool.F?

Reply:
Yes, am updating these. @jdmattern-noaa